tests/unit_tests/test_mig.py

```diff
@@ -3,7 +3,8 @@
 import pytest  # type:ignore[import]
 
 from maus.models.message_implementation_guide import SegmentGroupHierarchy, SegmentGroupHierarchySchema
-from unit_tests.serialization_test_helper import assert_serialization_roundtrip  # type:ignore[import]
+from tests.unit_tests.serialization_test_helper import assert_serialization_roundtrip  # type:ignore[import]
```

Comment:
```suggestion
from serialization_test_helper import assert_serialization_roundtrip  # type:ignore[import]
```
zumindest in tox und pycharm klappt das auch.

Reply:
#80